pkg/athena/api/api.go

```diff
@@ -307,6 +307,9 @@ func (c *API) Columns(ctx aws.Context, options sqlds.Options) ([]string, error)
 	for _, cat := range out.TableMetadata.Columns {
 		res = append(res, *cat.Name)
 	}
+	for _, par := range out.TableMetadata.PartitionKeys {
+		res = append(res, *par.Name)
```

Comment:
Would it be useful to add "(partitioned)" here like in Athena UI?

Reply:
Maybe, though I'll note that the Athena UI applies that suffix to the type and not to the column name (which I think is the correct choice).

Reply:
Understood. We can keep it without the additional string for now. 

Thank you for the contribution!